src/clients/checkout.ts

```diff
@@ -29,24 +65,33 @@ export class Checkout extends JanusClient {
     })
   }
 
+  private getRequestConfig(
```

Comment:
what do you think about making this method a util function? in that case, we could refactor the other clients' code to use it... passing also the context as a parameter. what do you think about this, @lucis?

Reply:
Good idea!